### ISSUE TITLE
label leaky

### ISSUE THREAD

**Issue by Vipermdl** [mksarker/SLSDeep#2]
Your validation set is the top 50 data in the test set in the ISBI2016 data set. Obviously, there is a problem with label leaks. I think your experiment results are not very accurate, or can you provide me with your complete code, thank you!

**Comment by mksarker** [comment]
This is an example for creating the .txt file. You can make your own label file for 2016 and 2017. I have used both dataset for the evaluation. Please follow the whole paper for training and test.  Thanks

**Comment by Vipermdl** [comment]
fine, thanks!
